py4DSTEM/process/utils/utils.py

```diff
@@ -548,8 +548,14 @@ def get_voronoi_vertices(voronoi, nx, ny, dist=10):
 
     return vertex_list
 
-
-def foo():
-    """Test func."""
-    print('pass')
+def ewpc_lambda(Q_Nx,Q_Ny):
```

Comment:
I would rename this - similar to my comment above, I don't see how being a lambda function is relevant.  Isn't the relevant info is that this is returning a function?  This name seems to obfuscate more than it helps - I'd recommend calling this `get_ewpc_filter_function` if you want to be explicit, or `get_ewpc_fn` if you want to be concise.